doc/usage/spark-demo-tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Running Spark Job using AppWrapper and MCAD Controller
+
+
+### Sources
+
+How-to run Spark in Kubernetes using CLI can be found in https://spark.apache.org/docs/latest/running-on-kubernetes.html#prerequisites 
+
+An example of using Spark Master service and workers can be found in https://github.com/kubernetes/examples/blob/master/staging/spark/README.md 
+
+The following example of running AppWrapper job for Spark is heavily based on the above examples/documentation.
+
+### Creating an AppWrapper job for Spark
+
```

Comment:
Can you explain what will be deployed in the AppWrapper?  Something like 1 Spark Master and 2 Workers and Client pod...?